Fix "by using" in concepts N--Z (#3207)
theforeman/foreman-documentation: guides/common/modules/con_resolving-package-dependencies.adoc

* When incrementally adding a security errata to a content view, dependency solving can cause significant delays to content view publication without major benefits.
* Packages from a newer erratum might have dependencies that are incompatible with packages from an older content view version.

> Incrementally adding the erratum by solving dependencies might include unwanted packages.

Reviewer: ```suggestion
Incrementally adding the erratum by solving dependencies might result in the inclusion of unwanted packages.
```
Maybe this.

[id="Combining_exclusion_filters_with_dependency_solving_{context}"]
.Combining exclusion filters with dependency solving
====

> You want to recreate {RHEL}{nbsp}8.3 by using content view filters and include selected errata from a later {RHEL}{nbsp}8 minor release.

Reviewer: "You want" is ambiguous. 

Are you saying that the user is required to do this? "Recreate RHEL 8.3  ..." or "You must recreate RHEL 8.3..."

Or is this an option? "You can recreate RHEL 8.3 ..."

Author: I think it's an example like "Let's say you want to...".

Author: ```suggestion
For example, you recreate {RHEL}{nbsp}8.3 by using content view filters and include selected errata from a later {RHEL}{nbsp}8 minor release.
```
